includes/compat/trait-wc-stripe-subscriptions.php

```diff
@@ -513,13 +522,24 @@ public function process_subscription_payment( $amount, $renewal_order, $retry =
 				throw new WC_Stripe_Exception( print_r( $response, true ), $localized_message );
 			}
 		} catch ( WC_Stripe_Exception $e ) {
-			WC_Stripe_Logger::log( 'Error: ' . $e->getMessage() );
+			WC_Stripe_Logger::error(
+				'Error processing subscription payment: ' . $e->getMessage(),
```

Comment:
Nit: just to be clear about the type.
```suggestion
				'Error processing subscription renewal payment: ' . $e->getMessage(),
```